.github/workflows/ci.yaml

```diff
@@ -201,7 +203,36 @@ jobs:
       arch: ${{ matrix.arch }}
       version: ${{ matrix.version }}
       runner-type: ${{ matrix.runner-type }}
-
+    macos-unit-test-coverage:
+    runs-on: macos-latest
```

Comment:
We should just add a `unit-test-coverage` step to [the existing `unit-tests` workflow](https://github.com/runfinch/finch/blob/9c72ad1dcc7829b2421ab8de07024201057dab1b/.github/workflows/ci.yaml#L80), instead of creating an entirely new workflow.

We also don't even need to use multiple runners for this, we can use the default `ubuntu-latest` and set GOOS as needed, per platform, like the linting workflow

Reply:
noted, this i meant to try out things only intention was the exclude, but if we have it in the unit test flow we probably dont need it will cancel this one.